test/test_manifest.json

```diff
@@ -1599,6 +1599,12 @@
        "lastPage": 6,
        "type": "eq"
     },
+    {  "id": "bug1661226",
+       "file": "pdfs/bug1661226.pdf",
+       "md5": "ff8328a52b48026342aedb5620f27e6d",
+       "rounds": 1,
+       "type": "eq"
```

Comment:
Doesn't this need to have `"forms": true` added as well here, to actually trigger the code-path that causes the problems?
As-is I'm not sure that you're actually testing what you want here, since the bug only manifests with `pdfjs.renderInteractiveForms = true` set.

Reply:
I've filed #12288 in regards to this.